.github/actions/install-anchor/action.yml

```diff
@@ -18,10 +14,9 @@ runs:
           ~/.cargo/registry/cache/
           ~/.cargo/git/db/
           ./target/
-        key: cargo-${{ runner.os }}-anchor-${{ hashFiles('**/Cargo.lock') }}
-    - run: cargo install --git ${{inputs.anchor_git}} anchor-cli --locked --force
+        key: anchor-cli-${{ runner.os }}-v0000-${{ env.ANCHOR_VERSION }}
+    - run:
+        cargo install --git https://github.com/project-serum/anchor --tag "v$ANCHOR_VERSION"
```

Comment:
Instead of requiring another param, my assumption was that we can always install from git? Can revert if that's not true.